src/www/ui/api/Controllers/JobController.php

```diff
@@ -68,6 +69,33 @@ class JobController extends RestController
    * @param array $args
    * @return ResponseHelper
    */
+
+  public function getAllJobs($request, $response, $args)
```

Comment:
Woops, looks like you messed the function documentation of `getAllJobs()` and `getJobs()`

Reply:
My bad!! Resolved it